src/app/services/authentication.service.ts

```diff
@@ -1,21 +1,68 @@
-import { Injectable, Inject } from '@angular/core';
+import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs/Observable';
+import { environment } from '../../environments/environment';
+
+import {EditUserBody, LoginBody, RegistrationBody} from '../models/body-obj.model';
```

Comment:
Please order the imports by the framework imports first, after application imports with a line separator between.